### DISCUSSION TITLE
Is the project still active?

### DISCUSSION THREAD

**Discussion by andrew-hu368** [stepci/stepci#117] [General]
Just wondering if StepCI is still under development.

**Comment by mishushakov** [comment]
Yes. We will be pushing an update in the next weeks and are still committed to the project.
Meanwhile, is there any feedback you could give us? ☺️

Let me know,
Thanks

**Reply by andrew-hu368** [reply]
Thank you for getting back to me. 

I love the project, but I had to implement some workarounds for asynchronous tasks.

For example, I upload a CSV file for later processing and get back an ID. After waiting for 5s, I want to check the job processing status. 
Currently, we are working around this limitation by running JS and using setTimeout. Unfortunately, this comes at the cost of being unable to capture and share data between tests.

**Reply by mishushakov** [reply]
So you want to have a pause between the steps, correct?

**Reply by andrew-hu368** [reply]
Pause or retry until a result is available.

**Reply by mishushakov** [reply]
That's a popular request, we will see what we can do 😉
I will reply here, If we have anything to share

Let me know if there's something else we could help you with
Thanks 🙏

**Comment by sebastianwis** [comment]
Hey Andrew @andrew-hu368 
Thank you for your contribution here! 
It would be amazing to welcome you to our Discord if you haven't joined it yet: 
https://discord.com/invite/KqJJzJ3BTu
This is where we keep users up to date with updates, features and bug resolutions.
